docs/user-docs.md

```diff
@@ -492,6 +492,28 @@ To use a specific Spark instance from the DC/OS Spark CLI:
 
     $ dcos config set spark.app_id <service.name>
 
+<a name="custom-docker-image"></a>
+# Custom Docker Image
+
+Note: custom docker images are not supported by Mesosphere.
+
+You can customize the Docker image in which Spark runs by extending
+the standard Spark Docker image. In this way, you can install your own
+libraries, such as a custom Python library.
+
+1. In your Dockerfile, extend from the standard Spark image and add you
+customizations:
+
+    ```
+    FROM mesosphere/spark:1.0.4-2.0.1
+    <custom commands>
+    ```
+
+1. Then build an image from your Dockerfile.
+
+1. Reference your custom Docker image with the `--docker-image` option
```

Comment:
Be specific:

`$ dcos spark run --docker-image=username/image:tag --submit-args="..."`